### DISCUSSION TITLE
新手，谁送我一点点币玩玩！

### DISCUSSION THREAD

**Discussion by hnzxb** [Chia-Network/chia-blockchain#1914] [Show and tell]
地址：xch1cf6ymad65j6s5sv7rphnjf3h0vtjwpwhys6l4ue3k2rd2klzekwqwvfprc
给点钱也可以的

**Comment by hoffmang9** [comment]
交易尚未启用。他们将在5月3日左右开始。

**Reply by wanghongqi8548488** [reply]
Are you sure? Can you transfer on May 3 rd

**Comment by wanghongqi8548488** [comment]
Are you sure? Can you transfer on May 3 rd
